docs/source/user/getting_started/advanced_workflow_concepts.rst

```diff
@@ -0,0 +1,309 @@
+.. jinja:: first_ctx
+
+Advanced workflow concepts
+###########################
+
+Resources
+----------
+
+Requesting resources can be done using a ``resources`` block, either for the whole workflow at the top level,
+
+.. code-block:: yaml
+
+    resources:
+    any:
+      scheduler: sge # Setting the scheduler is not normally needed because a
+                     # `default_scheduler` will be set in the config file.
+      scheduler_args:
+        shebang_args: --login
+        options:
+          -l: short
+
+
+or at the task level
+
+.. code-block:: yaml
+
+  - schema: simulate_VE_loading_damask
+    resources:
+      any:
+      # This will use two cores for input file generators and output file parsers
+      num_cores: 2
+      main:
+      # Use 16 cores for the "main" part of the task (the simulation in this case)
+      num_cores: 16
```

Comment:
```suggestion
      any:
        # This will use two cores for input file generators and output file parsers
        num_cores: 2
      main:
        # Use 16 cores for the "main" part of the task (the simulation in this case)
        num_cores: 16
```